lib/addStyles.js

```diff
@@ -102,14 +102,13 @@ function addStylesToDom (styles, options) {
 		if(domStyle) {
 			domStyle.refs++;
 
-			for(var j = 0; j < domStyle.parts.length && item.parts.length; j++) {
```

Comment:
@michael-ciniawsky I'm ashamed, a lot of issues :disappointed: Btw, we need add tests for this, someone in the future may want to do the same and repeat our mistake